specification/azuredata/resource-manager/Microsoft.AzureData/preview/2019-07-24-preview/azuredata.json

```diff
@@ -1984,6 +2272,88 @@
         }
       }
     },
+    "SqlServerInstanceProperties": {
+      "description": "Properties of SqlServerInstance.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "string",
+          "description": "null"
+        },
+        "edition": {
+          "type": "string",
+          "description": "null"
+        },
+        "containerResourceId": {
+          "type": "string",
+          "description": "null"
+        },
+        "createTime": {
+          "type": "string",
+          "description": "null"
+        },
+        "updateTime": {
+          "type": "string",
+          "description": "null"
+        },
+        "vCore": {
+          "type": "string",
+          "description": "null"
+        },
+        "status": {
```

Comment:
>"status" [](start = 8, length = 8)

what is status used for?  Is this a provisioningState?

Reply:
status can be "Connected" or "Disconnected". It is a Azure Arc resource type.